nrf_security/configs/psa_crypto_config.h.template

```diff
@@ -271,6 +271,8 @@
 #cmakedefine PSA_CRYPTO_DRIVER_ALG_SHA_384_OBERON               @PSA_CRYPTO_DRIVER_ALG_SHA_384_OBERON@
 #cmakedefine PSA_CRYPTO_DRIVER_ALG_SHA_512_OBERON               @PSA_CRYPTO_DRIVER_ALG_SHA_512_OBERON@
 #cmakedefine PSA_CRYPTO_DRIVER_ALG_STREAM_CIPHER_OBERON         @PSA_CRYPTO_DRIVER_ALG_STREAM_CIPHER_OBERON@
+#cmakedefine PSA_CRYPTO_DRIVER_ALG_TLS12_PRF_OBERON             @PSA_CRYPTO_DRIVER_ALG_TLS_PRF_OBERON@
```

Comment:
@PSA_CRYPTO_DRIVER_ALG_TLS_PRF_OBERON@ -> @PSA_CRYPTO_DRIVER_ALG_TLS12_PRF_OBERON@ 

The same for the next line @PSA_CRYPTO_DRIVER_ALG_TLS12_PSK_TO_MS_OBERON@

Reply:
Sharp eye, thank you. I'm wondering why this is not causing failures though...